rust/bitcoin_wallet/src/bitcoin_wallet/lib.rs

```diff
@@ -98,3 +118,45 @@ async fn get_fees() -> (Satoshi, Satoshi, Satoshi) {
         .unwrap();
     (current_fees[25], current_fees[50], current_fees[75])
 }
+
+/// Sends a transaction, transferring the specified Bitcoin amounts to the provided address.
+#[update]
+async fn transfer(
+    address: String,
+    amount: Satoshi,
+    fee: MillisatoshiPerByte,
+    allow_rbf: bool,
+) -> Result<TransactionInfo, MultiTransferError> {
+    let principal = get_authenticated_principal();
+    let principal_address = &get_principal_address(principal);
+    let address = Address::from_str(&address).unwrap();
+    let payouts = BTreeMap::from([(address, amount)]);
+
+    let get_utxos_args = BITCOIN_AGENT_USERS.with(|bitcoin_agent_users| {
+        bitcoin_agent_users.borrow_mut()[&principal]
+            .get_utxos_args(principal_address, MIN_CONFIRMATIONS)
+    });
+    let get_utxos_result = get_utxos_from_args(get_utxos_args).await.unwrap();
+    let multi_transfer_args = BITCOIN_AGENT_USERS.with(|bitcoin_agent_users| {
+        let mut bitcoin_agent_users_mut = bitcoin_agent_users.borrow_mut();
+        let bitcoin_agent = bitcoin_agent_users_mut.get_mut(&principal).unwrap();
+        bitcoin_agent.apply_utxos(get_utxos_result);
+        bitcoin_agent.get_balance_update(principal_address).unwrap();
+        bitcoin_agent.get_multi_transfer_args(
+            &payouts,
+            principal_address,
+            Fee::PerByte(fee),
+            MIN_CONFIRMATIONS,
+            allow_rbf,
+        )
+    });
+    let multi_transfer_result = multi_transfer_from_args(multi_transfer_args).await?;
```

Comment:
If transfer simply invokes multi_transfer with different parameters, we could also just call the functions `get_transfer_args` and `transfer_from_args`. But maybe it is cleaner this way. What do you think?

Reply:
The main problem with renaming `get_multi_transfer_args` to `get_transfer_args` is that we still need another function for `transfer` (in the sense of `mono_transfer` in comparison with `multi_transfer`), as `transfer` takes as argument `destination_address: &Address` and `amount: Satoshi` instead of `payouts: &BTreeMap<Address, Satoshi>` and needs the additional argument `is_fee_deduced_of_amount`. So in my opinion we can keep the current design or change `transfer` to `mono_transfer`, but it seems clear that naming `transfer` for both mono and multi transfer doesn't seem possible.
Of course later on when mono transfer will be available, the send bitcoins feature of the Bitcoin wallet should use this mono transfer function.